core/coreunix/add.go

```diff
@@ -10,19 +10,19 @@ import (
 	"path/filepath"
 	"strconv"
 
-	core "github.com/ipfs/go-ipfs/core"
+	"github.com/ipfs/go-ipfs/core"
```

Comment:
Is there a reason for these changes?  They are adding a lot of noise?

Reply:
Yeah,actually,it need not to rename the package name.The default name is also `core`,and renamed maybe redundant.

Reply:
There was a (short) discussion on this somewhere, the conclusion IIRC was that for consistency in some places we may actually prefer to have those labels. (some IDEs such as Goland do this by default, you can disable this in `Settings > Go > Imports > Optimize imports on the fly`)